<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CStageAddItem</class>
 <widget class="QWidget" name="CStageAddItem">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>568</width>
    <height>483</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>50</y>
     <width>111</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>StageName:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_stageName">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>50</y>
     <width>361</width>
     <height>35</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>120</y>
     <width>121</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>StartTime:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>190</y>
     <width>101</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>EndTime:</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_ok">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>310</y>
     <width>111</width>
     <height>38</height>
    </rect>
   </property>
   <property name="text">
    <string>ok</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_cancel">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>310</y>
     <width>111</width>
     <height>38</height>
    </rect>
   </property>
   <property name="text">
    <string>cancel</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_startTime">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>120</y>
     <width>361</width>
     <height>35</height>
    </rect>
   </property>
   <property name="placeholderText">
    <string/>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_endTime">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>190</y>
     <width>361</width>
     <height>35</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
